src/components/Card/Card.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300&display=swap');
+
+
+*,
```

Comment:
These 'reset' styles are a good thing to do but probably the best place for them is in the site's main stylesheet, or a 'reset' stylesheet, rather than in a component's stylesheet. That's because they'll apply to everything in the site. Only styles for the component should be in its stylesheet.

Reply:
Agreed, this is a repeat at component level so will leave this out here.